Timestamp,Level,Arquivo,Message
2025-07-24 14:36:38,INFO,base.py.start:181,Scheduler started
2025-07-24 14:36:38,INFO,schedule_compose.py.generate_base_scheduled:56,Atualizacao semanal agendada.
2025-07-24 14:36:38,INFO,base.py._real_add_job:895,"Added job ""generate_base_scheduled.<locals>.init"" to job store ""default"""
2025-07-24 14:36:38,INFO,schedule_compose.py.init:43,Start base generator.
2025-07-24 14:36:38,INFO,create_base_controller.py.create_bases:86,Skipping base generation
2025-07-24 14:36:38,INFO,schedule_compose.py.init:49,Finish base generator. time spent: 0:00:00.000281
2025-07-24 14:36:38,INFO,schedule_compose.py.init:53,skipping base Cache generation.
2025-07-24 14:36:51,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:36:52,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:36:55,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:36:56,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:02,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:04,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:07,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:08,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:12,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:13,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:17,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:18,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:23,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:24,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:28,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:29,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:34,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
2025-07-24 14:37:36,INFO,login_composer.py.login_composer:52,Nenhum provider foi capaz de logar o usuário.
2025-07-24 14:37:38,INFO,api_log.py._send_request:23,"<class 'src.infra.requests.api_log.ApiLog'> Falha no POST do log: 502, <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
"
